pages/community/tsc.js

```diff
@@ -72,6 +73,11 @@ export default function TSC() {
               can also build a great AsyncAPI-based project that we don't have
               yet in our GitHub organization and donate it (we'll ask you to
               stay as a maintainer).
+              Follow this
+              <TextLink href="https://www.youtube.com/watch?v=uG_aLF9Z1F0" target="_blank" className="text-base text-blue-500 hover:text-sky-400 no-underline">
```

Comment:
```suggestion
              <TextLink href="https://www.youtube.com/watch?v=uG_aLF9Z1F0" target="_blank" className="text-base font-normal text-blue-500 hover:text-sky-400 no-underline">
```

The font-weight of the text is getting large, so it should be made according to the remaining text in the paragraph.

Reply:
done..